app/cdap/components/GridTable/index.tsx

```diff
@@ -34,6 +34,8 @@ import { IExecuteAPIResponse, IRecords, IParams, IHeaderNamesList } from './type
 import { IValues } from 'components/WrangleHome/Components/OngoingDataExploration/types';
 import NoRecordScreen from 'components/NoRecordScreen';
 import T from 'i18n-react';
+import { ISnackbarToast } from 'components/ConnectionList/Components/TabLabelCanSample/types';
```

Comment:
can we check if this type is needed in the snackbar component types.ts so we can define the interface there and then reuse it here